ui.apps/src/main/content/jcr_root/apps/core/fd/components/formsportal/draftsandsubmissions/v1/draftsandsubmissions/draftsandsubmissions.html

```diff
@@ -0,0 +1,37 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2021 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+<sly data-sly-use.model="com.adobe.cq.forms.core.components.models.formsportal.DraftsAndSubmissions"
+     data-sly-use.logic="${'draftsandsubmissions.js' @ layout=model.layout}"
+     data-sly-use.clientlib="${'/libs/granite/sightly/templates/clientlib.html'}"
+     data-sly-use.items="${logic.layoutPath}"
+     data-sly-use.menu="${logic.menuPath}"></sly>
+<div class="cmp-draftsandsubmissions" data-cmp-is="draftsandsubmissions" id="${model.id}" data-queryPath="${resource.path}.model.json">
```

Comment:
Even this is incorrect, can you please fix the BEM selectors by analyzing all of them ?

Reply:
Updated class names
Please review